lib/parse.js

```diff
@@ -1,5 +1,5 @@
 /*jshint -W030 */
-var tagRE = /(?:<!--[\S\s]*?-->|<(?:"[^"]*"['"]*|'[^']*'['"]*|[^'">])+>)/g;
+var tagRE = /<[a-zA-Z\-\!\/](?:"[^"]*"|'[^']*'|[^'">])*>/g;
```

Comment:
```suggestion
var tagRE = /<[a-zA-Z0-9\-\!\/](?:"[^"]*"|'[^']*'|[^'">])*>/g;
```

Tags like `<0>`, `<1>` etc. are used by  [`react-i18next`](https://github.com/i18next/react-i18next).